src/Language/SystemVerilog/AST/Attr.hs

```diff
@@ -24,6 +25,14 @@ type AttrSpec = (Identifier, Expr)
 
 instance Show Attr where
     show (Attr specs) = printf "(* %s *)" $ commas $ map showSpec specs
+instance {-# OVERLAPPING #-} Show [Attr] where
+    show = foldr (\a b -> show a ++ " " ++ b) ""
```

Comment:
I would like to avoid adding usage of `FlexibleInstances` if possible. I think the precedent elsewhere in the AST module would be to define something like: `showAttrs :: [Attr] -> String`. This could be defined in `Expr.hs`, which would be the only user. I could also see this more succinctly written as `intercalate " " . map show` or similar.

Reply:
I forgot about `intercalate`. I'm going to do what you're suggesting.